test/tools/config_load_check/BUILD

```diff
@@ -12,7 +12,10 @@ envoy_package()
 envoy_cc_test_library(
     name = "config_load_check_lib",
     srcs = ["config_load_check.cc"],
-    deps = ["//test/config_test:config_test_lib"],
+    deps = [
+        "//source/common/config:protobuf_link_hacks",
```

Comment:
Ideally I don't think you really need the link hacks but the protobuf libraries behind it, though I think current build_fixer doesn't allow it happen without a bunch of `#include` statement. So I guess this is fine until we completely remove the link hacks.

Reply:
OK since @htuch is out this week let's just merge this and we can revisit if needed when he is back.